resources/gin/train/tracknet_v2_1_classifier_train.cfg

```diff
@@ -14,22 +14,22 @@ experiment.random_seed = 42
 
 
 ### data ###
-TrackNetV2_1DataLoader.batch_size = 32
-TrackNetV2_1DataLoader.train_dataset = @train/TrackNetV2_1Dataset
-TrackNetV2_1DataLoader.valid_dataset = @valid/TrackNetV2_1Dataset
-TrackNetV2_1DataLoader.max_size = 1000000
+EventWiseSplitDataLoader.batch_size = 32
+EventWiseSplitDataLoader.train_dataset = @train/TrackNetClassifierDataset
+EventWiseSplitDataLoader.valid_dataset = @valid/TrackNetClassifierDataset
+EventWiseSplitDataLoader.max_size = 100000
```

Comment:
maybe we can make this argument not obligatory and give it some default value

Reply:
same as for `nrows` in `parse.csv_params`

Reply:
It has default value, i used this in experiments. Now removed